skycfg_test.go

```diff
@@ -326,6 +346,24 @@ func TestSkycfgEndToEnd(t *testing.T) {
 				},
 			},
 		},
+		endToEndTestCase{
+			caseName:   "flatten nested list",
+			fileToLoad: "test13.sky",
+			expProtos: []proto.Message{
+				&pb.MessageV2{
```

Comment:
then add a `flattenLists` field to endToEndTestCase which passes `WithFlattenLists()` as an argument to `config.Main()` and set it to true here